### PR TITLE
P0533R9 constexpr for <cmath> and <cstdlib>

### LINKED ISSUE #5262: [2022-02 LWG Motion 3] P0533R9 constexpr for <cmath> and <cstdlib>
[P0533R9](https://wg21.link/p0533r9)

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #5262 
Fixes cplusplus/papers#104

### CODE REVIEW

File: source/lib-intro.tex [hunk 1]
```diff
@@ -136,6 +136,30 @@
...
+\begin{note}
+Annex F specifies
+the conditions under which floating-point exceptions are raised and
+the behavior when NaNs and/or infinities are passed as arguments.
+\end{note}
+\begin{note}
+Equivalently, a call to a C standard library function is
+a non-constant library call
+if \tcode{errno} is set
+when math \tcode{errhandling \& MATH ERRNO} is \tcode{true}.
```
Comment (JohelEGP) [inline source/lib-intro.tex:160]: This is how it looks in the paper.
```suggestion
when \tcode{math_errhandling \& MATH_ERRNO} is \tcode{true}.
```

File: source/numerics.tex [hunk 2]
```diff
@@ -9671,11 +9671,11 @@
...
-  float ceilf(float x);
-  long double ceill(long double x);
+  constexpr float ceil(float x);                // see \ref{library.c}
+  constexpr double ceil(double x);
+  constexpr long double ceil(long double x);    // see \ref{library.c}
+  constexpr float ceilf(float x);
+  constexpr long double ceill(long double x);
 
   float floor(float x);                 // see \ref{library.c}
   double floor(double x);
```
Comment (JohelEGP) [inline source/numerics.tex:9681]: Missing `constexpr` for these and the others 3 below them.
```suggestion
  constexpr float floor(float x);                 // see \ref{library.c}
  constexpr double floor(double x);
```

**Review by JohelEGP — CHANGES_REQUESTED** [review]

File: source/numerics.tex (hunk 2 above)
Comment (jensmaurer) [reply source/numerics.tex]: fixed

**Comment by tkoeppe** [comment]
@jensmaurer: is this missing the feature test macro?

**Comment by JohelEGP** [comment]
No, it's at https://github.com/cplusplus/draft/pull/5278/files#diff-f46371f476893d323cfcb1383e9a70f07ff1c1f0c217e22294d0bf937aae6794R592.

**Comment by tkoeppe** [comment]
Ah yes, thanks!

**Pull request merged**